source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -52,6 +52,27 @@ int HeaderMapWrapper::luaGet(lua_State* state) {
   }
 }
 
+int HeaderMapWrapper::luaGetAtIndex(lua_State* state) {
+  const char* key = luaL_checkstring(state, 2);
```

Comment:
Not sure if we can benefit by starting to use: https://github.com/envoyproxy/envoy/blob/e175358ee7e06e0292998a089b6ef4e893c1c509/source/extensions/filters/common/lua/lua.h#L79.

Reply:
Sure. I changed it for these methods. Happy to create a small follow-up PR to change it everywhere (don't wanna include too many other changes here)